db/migrate/20160525214823_promote_plant_parts_to_trait_descriptors.rb

```diff
@@ -0,0 +1,10 @@
+class PromotePlantPartsToTraitDescriptors < ActiveRecord::Migration
+  def change
+    add_reference :trait_descriptors, :plant_part, index: true
+    add_foreign_key :trait_descriptors, :plant_parts, on_delete: :nullify, on_update: :cascade
+
+    Rake::Task['curate:promote_plant_parts_to_traits'].invoke
+
+    remove_reference :plant_scoring_units, :plant_part
```

Comment:
While we are indeed dropping a column here, this doesn't have to be irreversible - the information is preserved in the DB and could be recovered. A `down` method could recreate the reference column and then find the required `plant_part` for each PSU by following `trait_scores.first.trait_descriptor.plant_part` before dropping the "new" relationship (i.e. `trait_descriptor` > `plant_part`).